packages/lab/src/tabs-next/TabstripNext.css

```diff
@@ -0,0 +1,68 @@
+/* Component class applied to the root element */
+.saltTabstripNext {
+  --tabs-tabstrip-display: var(--saltTabs-tabstrip-display, inline-flex);
+  --tabs-tabstrip-background: var(--saltTabs-tabstrip-background, transparent);
+
+  --tabs-activationIndicator-thumb-height: var(--saltTabs-activationIndicator-thumb-height, var(--salt-size-bar));
+  --tabs-activationIndicator-height: var(--saltTabs-activationIndicator-height, var(--salt-size-border));
+  --tabs-activationIndicator-background: var(--saltTabs-activationIndicator-background, var(--salt-container-primary-borderColor));
```

Comment:
It doesn't make sense as a border either, since there are two borders anyways, one for tabs and one for tab. Id rather keep it this way. It doesn't necessarily need to refer to a component.